SwrveSDK/src/google/java/com/swrve/sdk/config/SwrveConfig.java

```diff
@@ -62,4 +67,18 @@ public boolean isGAIDLoggingEnabled() {
     public void setGAIDLoggingEnabled(boolean enabled) {
         this.gAIDLoggingEnabled = enabled;
     }
+
+    /**
+     * @return if it will automatically obtain the Google Cloud Messaging Registration ID.
+     */
+    public boolean isObtainRegistrationIdEnabled() {
```

Comment:
Ok. Can't think of anything better, so automaticPushRegistration sounds good! :)